cartridge/twophase.lua

```diff
@@ -80,6 +89,26 @@ local function get_apply_config_timeout()
     return vars.options.apply_config_timeout
 end
 
+local function wait_config_release(timeout)
```

Comment:
Maybe we should log a warning while waiting?

Reply:
It's a critical section. We can't just log here because it could spam logs.